app/services/forest_liana/schema_adapter.rb

```diff
@@ -336,7 +336,8 @@ def add_enum_values_if_is_sti_model(column_schema, column)
 
     def sti_column?(column)
       (@model.inheritance_column &&
-       column.name == @model.inheritance_column) || column.name == 'type'
+        column.name == @model.inheritance_column) ||
+        (!@model.inheritance_column && column.name == 'type')
```

Comment:
STI parent that does not have children models should actually have their `inheritance_column` displayed even if it is the default `type` column.

Reply:
That is actually pretty smart, I'm gonna check your proposition